pkg/test/externalbinary/util.go

```diff
@@ -289,7 +226,7 @@ func createBinPath(path string) error {
 // an ImageStream object with tags associated with image-references from that payload.
 func extractReleaseImageStream(logger *log.Logger, extractPath, releaseImage string,
 	registryAuthFilePath string) (*imagev1.
-	ImageStream,
```

Comment:
verify job is failing on gofmt https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/29396/pull-ci-openshift-origin-master-verify/1876263943433883648 . May be this line.

Reply:
Unit test failures are because of https://issues.redhat.com/browse/OCPBUGS-47796

The lint failures are fixed though.